lib/request.rb

```diff
@@ -0,0 +1,22 @@
+require 'net/http'
+require 'uri'
+require 'json'
+require 'yaml'
+
+module Request
+  @config = YAML.load_file('config/config.yml')
+
+  def self.post(body)
+    uri = URI(@config['gist_api'])
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = (uri.scheme == "https")
+    req = Net::HTTP::Post.new(uri.path, 'Content-Type' => 'application/json')
+    req.body = body.to_json
+    puts req.body
```

Comment:
is the puts necessary to print?

Reply:
maybe make this smaller, move some part like building the body to another method